app/cdap/components/GridTable/index.tsx

```diff
@@ -289,6 +350,19 @@ export default function GridTable() {
           </TableBody>
         </Table>
       )}
+      {openDirectivePanel && (
+        <Box className={classes.directivePanelDiv}>
```

Comment:
styled component

Reply:
why do we need Box around the DirectiveInput
can we directly apply styles to DirectiveInput plz, make it styled component
and remove unnecessary Boxes or Wrappers unless very much necessary like has multiple children or so

here we don't need wrapper

Reply:
Done